frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -47,7 +47,7 @@ export const scene: SceneExport = {
 
 export function Surveys(): JSX.Element {
     const {
-        surveys,
+        surveys: { results: surveys },
```

Comment:
can we skip `results` here and add `surveys directly? its a bit strange that `surveys` is actually `surveys.results`